worker/worker.go

```diff
@@ -434,7 +434,9 @@ func (w *worker) withRevisionV3(ctx context.Context, contractID types.FileContra
 		return fmt.Errorf("%v: %w", "failed to acquire contract for funding EA", err)
 	} else {
 		defer func() {
-			if err := w.bus.ReleaseContract(ctx, contractID, lockID); err != nil {
+			releaseCtx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
I don't think we really need to. Goroutines are expensive and we should never really block here anyway because a worker that can't connect to the bus is quite broken anyway.